docs/source/faq.rst

```diff
@@ -110,7 +110,7 @@ Optuna saves hyperparameter values with its corresponding objective value to sto
 but it discards intermediate objects such as machine learning models and neural network weights.
 To save models or weights, please use features of the machine learning library you used.
 
-We recommend saving :obj:`~optuna.trial.Trial.trial_id` with a model in order to identify its corresponding trial.
+We recommend saving :obj:`optuna.trial.Trial.number` with a model in order to identify its corresponding trial.
```

Comment:
I think we should use ```:attr:`~optuna.trial.Trial.number` ``` (instead of `:obj: ...`) for generating an appropriate link.

Reply:
I conformed that `:obj:` works as well as `:attr:`. Sorry for my confusing comment.